src/loop_libraries/loop_library.cpp

```diff
@@ -0,0 +1,88 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#include <memory>
+
+#include "config.h"
+#include "loop_library.hpp"
+#include "loop_library_interface.hpp"
+
+#include "library_softsusy.hpp"
+
+#ifdef ENABLE_COLLIER
+#include "library_collier.hpp"
+#define COLLIER_INFO ", 1 (=COLLIER)"
+#else
+#define COLLIER_INFO
+#endif // ENABLE_COLLIER
+
+#ifdef ENABLE_LOOPTOOLS
+#include "library_looptools.hpp"
+#define LOOPTOOLS_INFO ", 2 (=LoopTools)"
+#else
+#define LOOPTOOLS_INFO
+#endif // ENABLE_LOOPTOOLS
+
+#ifdef ENABLE_FFLITE
+#include "library_fflite.hpp"
+#define FFLITE_INFO ", 3 (=fflite)"
+#else
+#define FFLITE_INFO
+#endif // ENABLE_FFLITE
+
+#define STRINGIFY(X) #X
+#define TOSTR(MACROS) STRINGIFY(MACROS)
+
+namespace flexiblesusy {
+
+int Looplibrary::type_ = -1;
```

Comment:
Can you please use an `enum` here instead of an `int` for readability?

Reply:
Yes, I am changing this.